federated-learning/c - Running Federated Computations in the Cloud.ipynb

```diff
@@ -35,7 +42,7 @@
     "import tensorflow as tf\n",
     "import tf_encrypted as tfe\n",
     "\n",
-    "config = tfe.RemoteConfig.load(players)\n",
+    "config = tfe.RemoteConfig.load(\"/tmp/config.json\")\n",
```

Comment:
this will not be OS-agnostic -- python has the `tempfile` module for doing this, see [here](https://stackoverflow.com/questions/847850/cross-platform-way-of-getting-temp-directory-in-python) for more

Reply:
Are we expecting people to run these examples with windows? If so we should probably come up with a plan to test everything on windows otherwise this should work fine on linux and macos but can also find a way to use `tempfile` or something similar main issue is that I'm not saving the file with python.

Reply:
This maybe shouldn't even go in a tmp directory, local file might be best.

Reply:
you're right on both counts; let's update the README to mention that we expect a UNIX-based laptop or for users to run the tutorials in the cloud

Reply:
Can do. Thought I saw somewhere where we were already mentioned that but I can't find it now.

Reply:
Think we should just leave the file in `/tmp/` though actually or some other standard place so that the scripts/notebooks can just look there instead of needing to copy config files around or anything.